.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - ubuntu-latest
           - macos-12
         python:
-          - 3.8
+          - 3.9
```

Comment:
```suggestion
          - 3.9
```
We can just remove the `- 3.8` here since the 3.9 options is already available.

Reply:
ooops -- missed that!  we just need to remove this line since followed by 3.9... will do, thanks

Reply:
rright, removal suggestion would be 
```suggestion
```